sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/LockRenewalOperation.java

```diff
@@ -183,14 +183,19 @@ public void close() {
         final OffsetDateTime now = OffsetDateTime.now();
         Duration initialInterval = Duration.between(now, initialLockedUntil);
 
-        if (initialInterval.isNegative()) {
-            logger.info("Duration was negative. now[{}] lockedUntil[{}]", now, initialLockedUntil);
+        // If remaining time to renew is less than 400ms, renew immediately
```

Comment:
Could you add a link to the class in c# in case we need to refer back to it again?

Reply:
Done! 👍